redisinsight/ui/src/pages/vector-search/VectorSearchPage.tsx

```diff
@@ -1,11 +1,22 @@
 import React from 'react'
+import { TelemetryPageView } from 'uiSrc/telemetry'
+import { usePageViewTelemetry } from 'uiSrc/telemetry/usePageViewTelemetry'
 
 import { VectorSearchCreateIndex } from './create-index/VectorSearchCreateIndex'
 import { VectorSearchQuery } from './query/VectorSearchQuery'
 
 export const VectorSearchPage = () => {
   const hasIndexes = false
 
+  usePageViewTelemetry({
```

Comment:
We can refactor all the existing places where we do copy/paste all the logic related to this "page view" tracking telemetry, and simply use this one liner. I'll create a separate ticket for that.